api/test/diag/consoleLogger.test.ts

```diff
@@ -127,6 +127,7 @@ describe('DiagConsoleLogger', () => {
 
       consoleFuncs.forEach(cName => {
         it(`should log ${fName} message even when console doesn't support ${cName} call before construction`, () => {
+          // @ts-ignore removing a console property is not allowed by types
```

Comment:
wouldnt it be better to use `ts-expect-error` here in case this is possible in the future for whatever reason ?